### ISSUE TITLE
Python implementation

### ISSUE THREAD

**Issue by cliffckerr** [andy-iskauskas/hmer#9]
Definitely one for the "not now" basket, but at some point in 2023 I'd love to see what ChatGPT could do with auto-translating `hmer` into Python :D

**Comment by andy-iskauskas** [comment]
Agreed, although my suspicion is that cleaning up the output would be as onerous as actually writing it in the first place...I daren't consider the possibility that it would write cleaner, more efficient code than me because that opens a whole existential can of worms!

On the subject of code implementation, it would be interesting to see how easy it is to convert visualisations made with `ggplot` to Python. That would probably be the major sticking point in converting the code, so it may be worth examining what Python packages best reproduce those visualisations. Any suggestions welcome!
